### ISSUE TITLE
Yaru broken with accent colors

### ISSUE THREAD

**Issue by rafaelmartines** [ubuntu/yaru#3632]
**Expected Behavior**

Gnome Shell follows the accent colors

**Actual Behavior**

I updated the system today and...

https://user-images.githubusercontent.com/8183691/166911079-98160db0-cae9-4fc3-8b97-d425682620e0.mp4

**Steps to Reproduce the Problem**

1. Go to Settings
2. Change do accent colors other than the orange one.

**System information**

Shell version: GNOME Shell 42.0
OS version:
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04 LTS
Release:	22.04
Codename:	jammy

**Yaru version**

Package: yaru-theme-gnome-shell
Version: 22.04.4
Priority: optional
Section: misc
Source: yaru-theme
Origin: Ubuntu
Maintainer: Ubuntu Desktop Team <ubuntu-desktop@lists.ubuntu.com>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 8.354 kB
Breaks: gnome-shell (<< 42~)
Homepage: https://community.ubuntu.com/c/desktop/theme-refresh
Task: ubuntu-desktop-minimal, ubuntu-desktop
Download-Size: 80,5 kB
APT-Manual-Installed: no
APT-Sources: http://archive.ubuntu.com/ubuntu jammy/main amd64 Packages
Description: Yaru GNOME Shell desktop theme from the Ubuntu Community
 This is the theme, better than a burrito, that is shaped by the community
 on the Ubuntu discourse.
 .
 This package contains the GNOME Shell theme part
 .
 If you want to follow development, more information here:
 https://community.ubuntu.com/c/desktop/theme-refresh.
 
 PS.: Also how can I contribute with the translation of this feature? there are a lot of things not translated in Brazilian Portuguese.

**Comment by ItzSwirlz** [comment]
You showed us gnome-software - that is a libadwaita app so I think that may mess up things...



> PS.: Also how can I contribute with the translation of this feature? there are a lot of things not translated in Brazilian Portuguese

@Jupi007 @Feichtmeier I would say make a translated version of the README, what do you guys think?

**Comment by Jupi007** [comment]
Hi @rafaelmartines,
thanks for the report.

I suppose the app you show is **Ubuntu software**?
If so, I think it is related to the fact it is a snap app.
With 22.04, we added many accent variants, but it looks like `gtk-common-themes` stable channel is still not shipping them.
That's why it revert to Adwaita, which is the default theme.

> PS.: Also how can I contribute with the translation of this feature? there are a lot of things not translated in Brazilian Portuguese.

What do you mean exactly?
You would like to translate the `README.md`?

**Comment by Feichtmeier** [comment]
sudo snap refresh gtk-common-themes --edge 
could help

CC @kenvandine

**Comment by rafaelmartines** [comment]
> Hi @rafaelmartines, thanks for the report.
> 
> I suppose the app you show is **Ubuntu software**? If so, I think it is related to the fact it is a snap app. With 22.04, we added many accent variants, but it looks like `gtk-common-themes` stable channel is still not shipping them. That's why it revert to Adwaita, which is the default theme.
> 

Yep is Ubuntu Software, but since the official released that problem didn't happened, this started again when I turned my notebook on today.

I'll try @Feichtmeier suggestion.

> > PS.: Also how can I contribute with the translation of this feature? there are a lot of things not translated in Brazilian Portuguese.
> 
> What do you mean exactly? You would like to translate the `README.md`?

For example, on the video I shared on main post, everything is in pt-br, but the accent colors is still "Color" instead ¨Cores" or "Destaque" in brazilian portuguese.

Another example, on Snap Store the "Everthing is up to date" is still in english instead brazilian portuguese.

**Comment by Jupi007** [comment]
> For example, on the video I shared on main post, everything is in pt-br, but the accent colors is still "Color" instead ¨Cores" or "Destaque" in brazilian portuguese.
> 
> Another example, on Snap Store the "Everthing is up to date" is still in english instead brazilian portuguese.

Honestly, I have no idea of how help in the translation.
You could maybe take a look at the repository of each app.

**Comment by rafaelmartines** [comment]
> > For example, on the video I shared on main post, everything is in pt-br, but the accent colors is still "Color" instead ¨Cores" or "Destaque" in brazilian portuguese.
> > Another example, on Snap Store the "Everthing is up to date" is still in english instead brazilian portuguese.
> 
> Honestly, I have no idea of how help in the translation. You could maybe take a look at the repository of each app.

No problem, I try to discover what the name of each app repository.

> sudo snap refresh gtk-common-themes --edge could help
> 
> CC @kenvandine

And It workd, closing the issue thanks 💓

**Comment by Jupi007** [comment]
@rafaelmartines Because there's still a problem for regular users, I reopen the issue so we can track the resolution progress :)
